job-server/src/spark.jobserver/JobServer.scala

```diff
@@ -66,6 +75,44 @@ object JobServer {
 
   }
 
+  private def storeInitialJars(config: Config, jarManager: ActorRef): Unit = {
+    if(config.hasPath("spark.jobserver.job-jar-uris")) {
+      val initialJarsConfig = config.getConfig("spark.jobserver.job-jar-uris").root
+
+      logger.info("Adding initial job jars: {}", initialJarsConfig.render())
+
+      val initialJars =
+        initialJarsConfig
+          .asScala
+          .map { case (key, value) => (key, value.unwrapped.toString) }
+          .toMap
+
+      // Ensure that the jars exist
+      for(jarPath <- initialJars.values) {
+        val f = new java.io.File(jarPath)
```

Comment:
So, the code here assumes that each of the strings in `job-jar-uris` is actually a local path, is that right?   I think this is an ok place to start, but maybe we should rename the config path as `job-jar-paths`?